rfd/0036/README.md

```diff
@@ -67,23 +118,141 @@ These types may include:
 
 Only `continuous` is required for the MVP.
 
-The types of compute providing these services may include:
+Service and compute types, are discussed further in the [services manifest](./services/manifest.md).
 
-- `docker` (default)
-- `infrastructure|lx|smartmachine`
-- `kvm|vm|hvm`
-- `manta`
 
-Only `docker`,`infrastructure`, and `kvm` are required for the MVP.
+### Project
 
-### Task queue
+While [services](#service) abstract any number of compute instances running the same software with the same configuration, a project allows users to group services and other resources together so that they can be managed as a whole without the distraction or complication or unrelated components.
+
+The concept of projects was first introduced with [RBACv2 in RFD13](../0013/README.md#proposal), this RFD intends to replace the definition of services from RFD13. Though projects and other features described in this RFD may be built independently of RBACv2 work, many assumptions about RBACv2 are made throughout this text.
```

Comment:
The requirements for what's implemented in projects and services are being defined here. RFD48 will need to be reviewed to ensure it offers the features to support usage described herein.